app/models/lookup_key.rb

```diff
@@ -7,6 +7,7 @@ class LookupKey < ActiveRecord::Base
 
   KEY_DELM = ","
   EQ_DELM  = "="
+  VALUE_REGEX = Regexp.new("\\A[\\w\\s]+#{EQ_DELM}[\\w\\s]+(#{KEY_DELM}[\\w\\s]+#{EQ_DELM}[\\w\\s]+)*\\Z")
```

Comment:
Second note, you might want to simply use `/\A... \z/` rather than Regexp.new, then you won't have to escape all of those slashes.